src/LogViewer/index.js

```diff
@@ -275,23 +277,29 @@ export default class LogViewer extends React.Component {
   }
 
   jumpToLine(lineNumber) {
-    const { minLineHeight, offset } = this.state;
+    const { offset, lineHeight } = this.state;
 
-    window.scrollTo(0, (lineNumber - offset) * minLineHeight + minLineHeight);
+    window.scrollTo(0, (lineNumber - offset) * lineHeight + 35);
     this.setState({ didLineJump: true });
   }
 
   jumpToQueriedLine(lineNumber) {
     const interval = setInterval(() => {
-      if (this.state.minLineHeight) {
+      if (this.state.chunkHeights.length) {
         this.jumpToLine(lineNumber);
         clearInterval(interval);
       }
     }, 1000);
   }
 
   render() {
-    const { isLoading, error, toolbarOpen, showLineNumbers, wrapLines, followLog } = this.state;
+    const { isLoading, error, toolbarOpen, showLineNumbers, wrapLines, followLog, lineHeight } = this.state;
+
+    const lineStyle = () => {
+      const height = lineHeight ? lineHeight : 19;
```

Comment:
No, and probably the easiest solution is to switch that constant to `let`. I'll comment there as well.